package.json

```diff
@@ -81,11 +81,11 @@
     "eslint": "^9.32.0",
     "eslint-config-love": "^121.0.0",
     "eslint-config-prettier": "^10.1.8",
-    "eslint-plugin-jsdoc": "^52.0.0",
+    "eslint-plugin-jsdoc": "^52.0.3",
```

Comment:
You've gone one higher [than this dependabot PR](https://github.com/ministryofjustice/laa-manage-your-civil-cases/pull/189/files
)


`"eslint-plugin-jsdoc": "^52.0.2",`

instead of

`"eslint-plugin-jsdoc": "^52.0.3",`